napari/utils/colormaps/colormaps.py

```diff
@@ -4,13 +4,13 @@
 import numpy as np
 from vispy.color import (
     BaseColormap,
-    Colormap,
     get_colormap,
     get_colormaps,
 )
 
 from ...types import ValidColormapArg
 from .vendored import cm, colorconv
+from .colormap import Colormap
 
 _matplotlib_list_file = os.path.join(
     os.path.dirname(__file__), 'matplotlib_cmaps.txt'
```

Comment:
in this PR or another one... can we just get rid of `matplotlibs_cmaps.txt`... I get the original inspiration, but it's the only place where we are reading a text file to create a list of strings.